app/jobs/concerns/hyrax/queued_job_behavior.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+module Hyrax
+  # Grants the user's edit access on the provided FileSet
+  module QueuedJobBehavior
+    extend ActiveSupport::Concern
+
+    included do
+      queue_as Hyrax.config.ingest_queue_name
+    end
+
+    private
+
+    def redis_queue
+      Valkyrie::IndexingAdapter.find(:redis_queue)
+    end
+
+    def requeue(*args)
+      self.class.set(wait_until: 5.minutes.from_now).perform_later(*args)
+    end
```

Comment:
The number of minutes (or seconds) to wait ought to be configurable. Or perhaps dynamic depending on the queue size?

Reply:
configurable seems reasonable, dynamic probably isn't needed.

Reply:
good idea. I've added it